test/unit/org/apache/cassandra/config/YamlConfigurationLoaderTest.java

```diff
@@ -50,4 +55,14 @@ public void fromMapTest()
         assertEquals(false, config.client_encryption_options.optional); // Check a nested object
         assertEquals(true, config.client_encryption_options.enabled); // Check a nested object
     }
+
+    @Test
+    public void sharedErrorReportingExclusions()
+    {
+        URL url = YamlConfigurationLoaderTest.class.getClassLoader().getResource("data/config/YamlConfigurationLoaderTest/shared_client_error_reporting_exclusions.yaml");
```

Comment:
This fails for me locally. Missing yaml file?

Reply:
ah, looks like ignore is kicking in

```
$ git add test/resources/data/
The following paths are ignored by one of your .gitignore files:
test/resources/data
hint: Use -f if you really want to add them.
hint: Turn this message off by running
hint: "git config advice.addIgnoredFile false"
```

I used the force (Luke)...